src/main/pages/setup-openshift/openshift-config.md

```diff
@@ -6,91 +6,42 @@ sidebar: user_sidebar
 permalink: openshift-config.html
 folder: setup-openshift
 ---
-## How It Works
-
-You can configure deployment of Che on OpenShift 3.6+ with env variables that are defined in [`che-config`](https://raw.githubusercontent.com/eclipse/che/master/deploy/openshift/che-config) file. If you have downloaded only deployment script but need to configure your Che server, download config file that should be located in the same directory with the deployment script.
-
-Once done, run the deployment script with `-c rollupdate` flag. Che deployment will be updated which automatically triggers a new deployment. You can also export envs in your environment, and the script will pick them up. It will only update Che server pod. This page focuses on some of the envs. You can either look at [Docker configuration page][docker-config] or [che.env](https://github.com/eclipse/che/blob/master/dockerfiles/init/manifests/che.env) file.
-
-We use recreate strategy. When the initial Che pod is shut down, server forcefully stops all workspace pods. Users working in those workspace will see a notification that the workspace is not running anymore. It does not impact data persistence - all workspace project files mounted through PVCs.
-
-## Update
-
-Export `CHE_IMAGE_TAG=${VERSION}` before executing `che_deploy.sh -c rollupdate`, where `${VERSION}` is a tag you want to update to. If you have your custom Che server image, you may export `CHE_IMAGE_REPO` too. You can get available tags on [Github](https://github.com/eclipse/che/tags) or [DockerHub](https://hub.docker.com/r/eclipse/che/tags/).
-
-If you want to always use the latest nightly builds, run `export IMAGE_PULL_POLICY=Always` and then re-run the deployment script. This way, OpenShift will always pull the image referenced in the deployment.
-
-Downgrades are possible, however, if database migrations have been previously performed, Che pod will fail to start.
-
-
-## OpenShift Flavor
-
-`OPENSHIFT_FLAVOR` defaults to `minishift`. Allowed values: `ocp`, `osio`
-
-## Username, Password, Token
-
-Deployment script needs access to a running OpenShift instance either via username/password or a token:
-
-```
-CHE_INFRA_KUBERNETES_USERNAME
-CHE_INFRA_KUBERNETES_PASSWORD
-CHE_INFRA_KUBERNETES_TOKEN
-```
-If token is set, it is used by `oc` client to log in and Che server to create namespaces and objects, else default credentials are used (`developer/developer`) for MiniShift and OCP.
 
-All objects that Che server creates such as workspace pods, services, routes and PVCs are created on behalf of this user. As a result, other OpenShift users who create workspaces in a multi-user Che, have no access to these objects.
+## Admin Guide
 
-This is going to be changed once [this issue](https://github.com/eclipse/che/issues/8178) is completed.
+This page describes OpenShift specific configuration. Refer to [Kubernetes Admin Guide][kubernetes-admin-guide] to general information that works both for OS and K8S.
```

Comment:
So for OpenShift someone would do everything in the K8S section, then come back here and do all the stuff here?

Reply:
Fixed. Now k8s guide is embedded in k8s and openshift admin guide pages